README.md

```diff
@@ -1539,7 +1540,7 @@ Print message(s) in green do:
 prompt.ok(...)
 ```
 
-#### 2.12.2 warn
+#### 2.11.2 warn
```

Comment:
Minor edit to the readme - this should be done regardless of if this PR goes through.